locale/fi/home.json

```diff
@@ -2,7 +2,7 @@
   "title": "Etusivu",
   "intro.title": "Suomi.fi Design System – digipalvelujen yhteiset rakennuspalikat",
   "intro.description": "Suomi.fi Design System tarjoaa kokoelman uudelleenkäytettäviä ja dokumentoituja käyttöliittymäkomponentteja. Kaikki komponentit on toteutettu ja testattu saavutettavuusvaatimusten mukaisesti. ",
-  "note.title": "Tämä on Suomi.fi Design Systemin julkinen kehitysversio (beta)",
+  "note.title": "Tämä on Suomi.fi Design Systemin julkinen kehitysversio",
```

Comment:
Is this still a development version? I think that was referring to public beta.

Reply:
Ah, this disappeared from the page along with the lower half of the footer, but the content still remained in the locale files. I'll remove it from there as well.